src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/extension/BuildCardExtension/build-card-script-helpers.js

```diff
@@ -0,0 +1,7 @@
+Handlebars.registerHelper('startScript', function() {
+    return new Handlebars.SafeString("<scr" + "ipt>");
```

Comment:
@shlomomdahan I don't see the need to separate `<scr` and `ipt>` here. That made sense when this was inline, where `<script>` tags were permissible, but now that it has been extracted this is no longer possible and it would be more readable to combine the two strings together.

Reply:
Agreed - I deleted the file.